web/splash/splash.js

```diff
@@ -2,10 +2,15 @@ import './splash.css';
 import newExpensifyLogo from 'logo?raw';
 import themeColors from '../../src/styles/themes/default';
 
+let areFontsReady = false;
+document.fonts.ready.then(() => {
```

Comment:
According to https://caniuse.com/mdn-api_document_fonts this works on everything except IE, which is officially EOL as of Summer 2022, so I really don't think we need to continue supporting it.